app/views/wiki/_header.html.erb

```diff
@@ -31,7 +31,7 @@
 
     <div style="padding-top:8px;text-align:center">
       <span class="d-none d-lg-inline">
-        <i style="color:#888;" class="fa fa-user"></i> <%= number_with_delimiter(@node.authors.length)  %>  |
+        <span rel="tooltip" title="The number of authors for this page." data-placement="top"><i style="color:#888;" class="fa fa-user"></i> <%= number_with_delimiter(@node.authors.length) %></span>  ||
```

Comment:
I think you may have added an extra vertical bar ("|") after the tooltip.

Reply:
I think there should only be one.

Reply:
@anthony-zhou good catch. There should be only one vertical bar.